app/src/main/kotlin/com/wire/android/WireApplication.kt

```diff
@@ -198,8 +199,14 @@ class WireApplication : Application(), Configuration.Provider {
             .build()
 
         val credentials = Credentials(clientToken, environmentName, appVariantName, applicationId)
+        val extraInfo = mapOf(
+            Pair("encrypted_proteus_storage_enabled", runBlocking {
+                globalDataStore.isEncryptedProteusStorageEnabled().first()
+            })
```

Comment:
Just some syntax sugar
```suggestion
        val extraInfo = mapOf(
           "encrypted_proteus_storage_enabled" to runBlocking {
                globalDataStore.isEncryptedProteusStorageEnabled().first()
            }
```